### PR TITLE
Convert Azure SecurityGroup to V2Wizard

### PR DESCRIPTION
**Pull request by amandabaker**

### CODE REVIEW

**Comment by anotherchrisberry** [comment]
looks like a test is failing...

**Comment by JargoonPard** [comment]
@anotherchrisberry PTAL

**Comment by duftler** [comment]
Pretty sure Chris is away. @zanthrash are you able to review this please?

**Comment by JargoonPard** [comment]
That explains it. Thanks @duftler

File: app/scripts/modules/azure/securityGroup/configure/CreateSecurityGroup.controller.spec.js [hunk 1]
```diff
@@ -12,14 +12,14 @@
...
-     beforeEach(window.inject(function ($controller, $rootScope, $q, accountService, securityGroupReader, modalWizardService,
+     beforeEach(window.inject(function ($controller, $rootScope, $q, accountService, securityGroupReader, v2modalWizardService,
                                  taskMonitorService, azureSecurityGroupWriter) {
 
        this.$scope = $rootScope.$new();
        this.$q = $q;
        this.accountService = accountService;
        this.securityGroupReader = securityGroupReader;
-       this.modalWizardService = modalWizardService;
+       this.modalWizardService = v2modalWizardService;
```
Comment (anotherchrisberry) [inline app/scripts/modules/azure/securityGroup/configure/CreateSecurityGroup.controller.spec.js]: AFAICT, this service isn't used or injected by the `azureCreateSecurityGroupCtrl` - if you remove this line and the parameter in the `inject` call above, do these tests still pass?

File: app/scripts/modules/azure/securityGroup/configure/CreateSecurityGroupCtrl.js [hunk 2]
```diff
@@ -9,7 +9,7 @@ module.exports = angular
     require('../securityGroup.write.service.js'),
     require('../../../core/region/regionSelectField.directive.js'),
     require('../../../core/account/account.service.js'),
-    require('../../../core/modal/wizard/modalWizard.service.js'),
+    require('../../../core/modal/wizard/v2modalWizard.service.js'),
```
Comment (anotherchrisberry) [inline app/scripts/modules/azure/securityGroup/configure/CreateSecurityGroupCtrl.js]: From what I see in the code, you _should_ be able to remove this line altogether...

**Comment by anotherchrisberry** [comment]
I'd try removing the references above to `v2modalWizardService` and verify everything works and there are no errors in the console when the modal opens. If that's the case, LGTM - just needs a rebase and can be merged.

Ping when you're ready for it to be merged and we'll get it in right away so you don't have to keep rebasing against new changes.

**Comment by amandabaker** [comment]
@anotherchrisberry PTAL

**Pull request merged**